lib/Watching.js

```diff
@@ -74,6 +74,7 @@ class Watching {
 					});
 				});
 			};
+			this.compiler.watchMode = true;
```

Comment:
This is set a bit too late. There is a `this.compiler.running` flag. Set `watchMode` to true when this is set. And set it to false when it's set to `false` in `Watching.close`